<?xml version="1.0"?>
<launch>
	<param name="target_frame" type="string" value="/zed2/zed_left_camera"/>
	<param name="source_frame" type="string" value="/centerlaser"/>
	<node pkg="sensor_fusion" type="coloringcloud" name="coloring_zed2">
		<remap from="/cloud" 		 to="/sq_lidar/points/left" />
		<remap from="/camera_info" 	 to="/zed2/left/camera_info" />
		<remap from="/image" 		 to="/zed2/left/image_rect_color/republish" />
		<remap from="/cloud/colored" to="/zed2/colored" />
	</node>
</launch>
